beam/hooks.py

```diff
@@ -241,3 +246,9 @@
 # auth_hooks = [
 # 	"beam.auth.validate"
 # ]
+
+from erpnext.controllers.subcontracting_controller import SubcontractingController
+
+from beam.beam.overrides.subcontracting_controller import update_stock_ledger
+
+SubcontractingController.update_stock_ledger = update_stock_ledger
```

Comment:
Please no overrides inside of hooks.py. We should be using the doctype override in a case like this.

Reply:
I have removed monkey patch and added as doctype base class override